.github/workflows/ruby.yml

```diff
@@ -1,77 +1,21 @@
 name: Ruby tests
+
 on:
   pull_request:
-env:
-  BUNDLE_WITHOUT: journald:development:console:libvirt
-  RAILS_ENV: test
-  DATABASE_URL: postgresql://postgres:@localhost/test
-  DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: true
+
 concurrency:
   group: ${{ github.head_ref }}
   cancel-in-progress: true
+
 jobs:
   rubocop:
     uses: theforeman/actions/.github/workflows/rubocop.yml@v0
     with:
       command: bundle exec rubocop --parallel --format github
```

Comment:
Is the `command` needed? Checking [the Readme](https://github.com/stejskalleos/foreman_actions/tree/v0#rubocop), it should work without it.

Reply:
The difference is that one uses the Rake task, but this has 2 benefits: parallel execution (faster) and it adds any failures as annotations in GitHub. You can add the formatter in the rake task (autodetecting the GHA environment), but there's no way for the parallel part to be there.

So you can argue about benefits in either direction. As the maintainer of the repository I'll put it with you.

For what it's worth, I've been considering moving the GHA to this exact command as a default, without detecting a Rakefile. It's been a long time since rubocop has introduced the standalone command and by now we should probably just force any repositories to update.